test/service_binding_test/service_binding_test.go

```diff
@@ -675,6 +677,115 @@ var _ = DescribeTestsFor(TestCase{
 								})
 
 								if testCase.async {
+									When("maximum polling duration is reached while polling", func() {
+										var oldCtx *TestContext
+										BeforeEach(func() {
+											brokerID, brokerServer, servicePlanID = newServicePlanWithMaxPollingDuration(ctx, true, MaximumPollingDuration)
+											brokerServer.ShouldRecordRequests(false)
+											EnsurePlanVisibility(ctx.SMRepository, TenantIdentifier, types.SMPlatform, servicePlanID, TenantIDValue)
+											resp := createInstance(ctx.SMWithOAuthForTenant, false, http.StatusCreated)
+											instanceName = resp.JSON().Object().Value("name").String().Raw()
+											Expect(instanceName).ToNot(BeEmpty())
+
+											postBindingRequest = Object{
+												"name":                "test-binding",
+												"service_instance_id": instanceID,
+											}
+											syncBindingResponse = Object{
+												"async": false,
+												"credentials": Object{
+													"user":     "user",
+													"password": "password",
+												},
+											}
+
+											oldCtx = ctx
+											ctx = NewTestContextBuilderWithSecurity().WithEnvPreExtensions(func(set *pflag.FlagSet) {
+												Expect(set.Set("operations.action_timeout", ((MaximumPollingDuration + 5) * time.Second).String())).ToNot(HaveOccurred())
+											}).BuildWithoutCleanup()
+
+											brokerServer.BindingHandlerFunc(http.MethodPut, http.MethodPut+"1", ParameterizedHandler(http.StatusAccepted, Object{"async": true}))
+											brokerServer.BindingLastOpHandlerFunc(http.MethodPut+"1", ParameterizedHandler(http.StatusOK, Object{"state": "in progress"}))
+										})
+
+										AfterEach(func() {
+											ctx = oldCtx
+										})
+
+										When("orphan mitigation unbind synchronously succeeds", func() {
```

Comment:
what about async orphan mitigation ? i think there were async orphan mitigation tests somewhere in the thousands of lines